<launch>
    <node name="ypspur_ros" pkg="ypspur_ros" type="ypspur_ros" output="screen">
        <param name="_compatible" value="1"/>
        <param name="port" value="/dev/ttyACM1"/>
        <param name="param_file" value="/home/deguchi-a/researches/programs/platform/yp-robot-params/robot-params/beego.param"/>
    </node>
    <node name="demopro2023" pkg="yamasemi2023_follow" type="demopro2023" output="screen">
        <param name="param_file" value="/home/deguchi-a/researches/programs/platform/yp-robot-params/robot-params/beego.param"/>
    </node>

    <include file="$(find arduino_dynamixel)/launch/box_controller.launch"/>
</launch>
